packages/FormElement/stories/examples/Showcase.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { boolean, select, text } from "@storybook/addon-knobs";
+import Heading from "@paprika/heading";
+import Input from "@paprika/input";
+import { Rule, Tagline } from "storybook/assets/styles/common.styles";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import { FormElementStory } from "../FormElement.stories.styles";
+import FormElement from "../../src";
+
+const formElementProps = () => ({
+  hasOptionalLabel: boolean("hasOptionalLabel", false),
+  hasRequiredLabel: boolean("hasRequiredLabel", false),
+  id: text("id", "input-id"),
+  isDisabled: boolean("isDisabled", false),
+  isInline: boolean("isInline", false),
+  isLabelVisuallyHidden: boolean("isLabelVisuallyHidden", false),
+  isReadOnly: boolean("isReadOnly", false),
+  label: text("label", "Form element"),
+  size: select("size", ShirtSizes.DEFAULT, "medium"),
+});
+
+const errorPropKnobs = () => ({
+  showError: boolean("show error", true),
```

Comment:
I see now what you're trying to do with this... but I think it might be better leave this out, and allow the viewer to discover how the component works. This suggests there's a prop called `showError`, which is not the case.

Reply:
Maybe for the showcase it makes more sense for the `errorText` to be blank by default.